Jenkinsfile

```diff
@@ -21,7 +21,7 @@ def secrets = [
   ]
 ]
 
-String dockerCompose = 'docker-compose -f integration-tests/docker-compose.yml -f docker-compose.yml --project-directory .'
+String dockerCompose = "docker-compose -f integration-tests/docker-compose.yml -f docker-compose.yml --project-name ${env.BUILD_TAG}"
```

Comment:
rafal do you remember the reason for setting the project directory?

Reply:
I recall I set it because otherwise it would take `integration-tests` for the project root directory, not the current one.